.github/workflows/update-images.yaml

```diff
@@ -286,8 +313,12 @@ jobs:
     - name: Commit
       run: |
         # git add returns success for files that exist and haven't been modified
-        git add kctf-operator/pkg/resources/constants.go
+        git add kctf-operator/resources/constants.go
+        git add kctf-operator/operator.yaml
         git add dist/resources/operator.yaml
```

Comment:
why are there two operator.yaml?

Reply:
`kctf-operator/operator.yaml` is generated by the Makefile and it is moved to `kctf-operator/operator.yaml` for release by this script. the only one that matters is `dist/resources/operator.yaml` - we could `.gitignore` the one on `kctf-operator` though, to avoid duplication

Reply:
added to gitignore